### PR TITLE
(improvement) settings page: added form for Paper Trading API keys

### PR DESCRIPTION
**Pull request by Blaumaus**
ASANA Ticket: [[Settings][API Keys] - Allow users to edit both the API keys and paper trading keys regardless of the mode currently active](https://app.asana.com/0/1125859137800433/1200101612219001/f)

UI Demonstration:
bfx-hf-ui-api-creds

### CODE REVIEW

File: src/pages/Settings/Settings.js [hunk 1]
```diff
@@ -230,3 +259,22 @@ export default class Settings extends React.PureComponent {
...
+  getActiveAOs: PropTypes.func.isRequired,
+  submitAPIKeys: PropTypes.func.isRequired,
+  updateSettings: PropTypes.func.isRequired,
+  gaUpdateSettings: PropTypes.func.isRequired,
+}
+
+Settings.defaultProps = {
+  ga: null,
+  dms: null,
+  activeExchange: 'bitfinex',
```
Comment (bhoomij) [inline src/pages/Settings/Settings.js:277]: can use DEFAULT_EXCHANGE

File: src/pages/Settings/Settings.js [hunk 2]
```diff
@@ -88,6 +88,10 @@ export default class Settings extends React.PureComponent {
       this.onSubmitAPIKeys(this.state)
     }
 
+    if (paperApiKey.trim().length && paperApiSecret.trim().length) {
```
Comment (bhoomij) [inline src/pages/Settings/Settings.js:91]: please use _trim/_size lodash helpers

**Review by bhoomij — CHANGES_REQUESTED** [review]

File: src/redux/reducers/ui/index.js [hunk 3]
```diff
@@ -11,8 +11,8 @@ const ACTIVE_MARKET_KEY = 'HF_UI_ACTIVE_MARKET'
 const ACTIVE_MARKET_PAPER_KEY = 'HF_UI_PAPER_ACTIVE_MARKET'
 const ACTIVE_EXCHANGE_KEY = 'HF_UI_ACTIVE_EXCHANGE'
 const IS_PAPER_TRADING = 'IS_PAPER_TRADING'
-const PAPER_MODE = 'paper'
-const MAIN_MODE = 'main'
+export const PAPER_MODE = 'paper'
+export const MAIN_MODE = 'main'
 
 const DEFAULT_ROUTE = '/'
 const DEFAULT_EXCHANGE = 'bitfinex'
```
Comment (bhoomij) [inline src/redux/reducers/ui/index.js:18]: we can export DEFAULT_EXCHANGE to use as default prop-type for activeExchange

**Review by alexstotsky — APPROVED** [review]
lgtm

**Review by bhoomij — APPROVED** [review]

**Pull request merged**